src/main/java/seedu/address/model/Model.java

```diff
@@ -83,5 +84,6 @@ public interface Model {
      * Updates the filter of the filtered person list to filter by the given {@code predicate}.
```

Comment:
The javadoc says `filter by the given {@code predicate}` but the variable is now changed to personPredicate
And the next line tagPredicate, which you can also add another javadoc for

Reply:
Oo initially that JavaDoc was supposed to be for both `updateFilteredPersonList `and `updateFilteredPersonListByTag` but I now realise I should separate the JavaDocs. Thanks!